dev/tests/integration/testsuite/Magento/Bundle/_files/order_with_bundle_shipped_together.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\Catalog\Model\Product\Type\AbstractType;
+use Magento\TestFramework\Workaround\Override\Fixture\Resolver;
+
+Resolver::getInstance()->requireDataFixture('Magento/Catalog/_files/products.php');
+
+/** @var $objectManager \Magento\TestFramework\ObjectManager */
```

Comment:
Type should be first in @var declaration

Reply:
updated